CMakeLists.txt

```diff
@@ -59,3 +59,11 @@ enable_testing()
 
 add_subdirectory(tests)
 
+#STARTUP MEX-TOOLBOX
+# The following line is to properly configure the installation script of the Mex-toolbox
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/startup_Mex.m.in ${CMAKE_BINARY_DIR}/startup_Mex.m)
+# Install configuration files
+install(FILES ${CMAKE_BINARY_DIR}/startup_Mex.m DESTINATION ${CMAKE_INSTALL_PREFIX}/mex)
```

Comment:
Furthermore, `startup_Mex.m` is a bit confusing name , as `mex` is a generic suffix. `startup_mexwholebodymodel.m` perhaps could be a better name.